{% extends "base.html" %}
  
{% block title %}References for President's Database{% endblock %}

{% block page_content %}
<div class="page-header">
    <h1> References for the Project: </h1>
    <h3> Research Articles </h3> <br>
    <p> Sagulenko, P., Puller, V., & Neher, R. A. (2018). TreeTime: Maximum-likelihood phylodynamic analysis. Virus Evolution, 4(1), vex042. https://doi.org/10.1093/ve/vex042 </p>
    <p> Cardona-Ospina, J. A., Rojas-Gallardo, D. M., Garzón-Castaño, S. C., Jiménez-Posada, E. V., & Rodríguez-Morales, A. J. (2021). Phylodynamic analysis in the understanding of the current COVID-19 pandemic and its utility in vaccine and antiviral design and assessment. Human Vaccines & Immunotherapeutics, 17(8), 2437–2444. https://doi.org/10.1080/21645515.2021.1880254 </p>
    <p> Attwood, S. W., Hill, S. C., Aanensen, D. M., Connor, T. R., & Pybus, O. G. (2022). Phylogenetic and phylodynamic approaches to understanding and combating the early SARS-CoV-2 pandemic. Nature Reviews. Genetics, 23(9), 547–562. https://doi.org/10.1038/s41576-022-00483-8 </p>
    <p> Phylodynamic analysis. (2020, January 29). Virological. https://virological.org/t/phylodynamic-analysis-176-genomes-6-mar-2020/356 </p>
    <p> Seemann, T. (2014). Prokka: rapid prokaryotic genome annotation. Bioinformatics (Oxford, England), 30(14), 2068–2069. https://doi.org/10.1093/bioinformatics/btu153 </p>
        
    <h3> Software libraries</h3>
    <p> Prokka, Biopython, Python libraries.</p> <br>
</div>
{% endblock %}
~                
